drivers/display/Kconfig.st7789h2

```diff
@@ -0,0 +1,32 @@
+# ST7789H2 display driver configuration options
+
+# Copyright (c) 2021, Jeremy LOCHE
+# SPDX-License-Identifier: Apache-2.0
+
+config ST7789H2
+	bool "ST7789H2 display driver"
+	depends on GPIO
+	help
+	  Enable driver for ST7789H2 display driver.
+
+config ST7789H2_GPIO_PORT_WRITING
+	bool "GPIO_PORT_WRITING"
```

Comment:
Please give this a proper title and not just part of the settings name.

Reply:
Sounds OK to me, there was the help section describing what it does but I'll provide a better title for this 👍